go.mod

```diff
@@ -206,3 +206,5 @@ replace github.com/pingcap/tidb/parser => ./parser
 
 // fix potential security issue(CVE-2020-26160) introduced by indirect dependency.
 replace github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
+
+replace github.com/tikv/client-go/v2 => github.com/hawkingrei/client-go/v2 v2.0.0-alpha.0.20220617031655-cee15ffb037e
```

Comment:
Why were the path of client-go changed?

Reply:
it just add some log to debug the test problem. it  will be removed when to release and also have some debug log to be removed.